### ISSUE TITLE
Tree selection types and AT - Ensure content is visible to assistive technologies - (2036570422)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5619]
### Module: 
 38 Tree and Tree Item 

### Violation: 
 Ensure content is visible to assistive technologies on the testing page for [Test 3: Selection mode](https://geospatialem.github.io/calcite-samples/tree.html) for the `"Children"` and `"Multi-children"` selection modes.

![image](https://user-images.githubusercontent.com/5023024/202571459-091a25a5-d221-4985-942a-13ec064df4ee.png)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036570422 

### Severity: 
 10 

### Description: 
 [Issue]
There are elements that are available on expanding controls that are hidden for assistive technology users. Example included:

- Child 2
- Child 3
- Child 2 (expanded by default)

[User Impact]
Sighted keyboard-only users will not understand what this element is, and will perceive it as an unnecessary tab stop.

[Code Reference] 
 ```
<calcite-tree slot="children" child="" scale="s" selection-mode="single" calcite-hydrated="" tabindex="-1">
  <calcite-tree-item aria-hidden="false" role="treeitem" tabindex="0" depth="2" scale="s" calcite-hydrated="">Grandchild 1</calcite-tree-item>
  <calcite-tree-item aria-expanded="false" aria-hidden="false" role="treeitem" tabindex="0" depth="2" has-children="" scale="s" calcite-hydrated="">Grandchild 2<calcite-tree slot="children" tabindex="-1" child="" scale="s" selection-mode="single" calcite-hydrated="">
    <calcite-tree-item aria-hidden="false" role="treeitem" tabindex="0" depth="3" scale="s" calcite-hydrated="">Great-grandchild 1</calcite-tree-item>
    (...)
  </calcite-tree-item>
</calcite-tree>
 ``` 

### Note: 
 Recommendation]
Developer should ensure that content is available for all assistive technology users. 
 

### Media Type: 
 Page Structure 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
@saralawrence Can you test this one for us? Unable to duplicate with JAWS, as it seems to be coming in as-expected.

**Comment by SaraLawrence** [comment]
@geospatialem What the tester wrote makes no sense to me. The main issue is with the keyboard commands not following the expected pattern (tabbing to each tree node as a group, using down arrow to move to next same level item, right arrow to expand when grandchildren exist, left arrow to close, up arrow to move up to next same level item, etc.) Also, tab from any level to move on to next tree node or other focusable item.

I would not spend time trying to fix this as most of the documentation I've looked at recommends NOT trying to make treeviews accessible [.](https://www.w3.org/WAI/ARIA/apg/example-index/treeview/treeview-navigation) 

A good alternative is the disclosure pattern [.](https://www.w3.org/WAI/ARIA/apg/example-index/disclosure/disclosure-navigation.html) 

Would it be possible to make note of the issues with accessibility that exist for tree widgets and direct people to use the disclosure pattern or a navigation menubar instead?

**Comment by geospatialem** [comment]
Thanks much for taking a look!

> Would it be possible to make note of the issues with accessibility that exist for tree widgets and direct people to use the disclosure pattern or a navigation menubar instead?

@SaraLawrence Totally agree, how does our [current documentation](https://developers.arcgis.com/calcite-design-system/components/tree/#keyboard-navigation) look?

**Comment by geospatialem** [comment]
This is not a bug, but could contribute to some documentation updates. Closing here in favor of updating the documentation separately.
